<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JoyStream</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addTorrentFilePushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>add torrent file</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="earnedBalanceLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>16</y>
      <width>100</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="earnedTitleQlabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>44</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>earned</string>
    </property>
   </widget>
   <widget class="QLabel" name="spentTitleLabel">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>44</y>
      <width>35</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>spent</string>
    </property>
   </widget>
   <widget class="QLabel" name="spentBalanceLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>16</y>
      <width>100</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>954</x>
      <y>16</y>
      <width>100</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="balanceTitleLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>44</y>
      <width>45</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusMessageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>791</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="unconfirmedBalanceLabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>16</y>
      <width>100</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="unconfirmedBalanceTitleLabel">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>44</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>unconfirmed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addMagnetLinkPushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>15</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>add magnet link</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QTreeView" name="torrentsTreeView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>691</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
